resources/home/dnanexus/eggd_artemis.py

```
@@ -0,0 +1,438 @@
+                            f'=HYPERLINK("{cnv_session_url}",'
+                            f' "{cnv_session_url}")'
+                        ),
+                        "cnv_excluded_regions_df": cnv_excluded_regions_df,
+                    }
+                )
+
+    return outputs
+
+
+@dxpy.entry_point("main")
+def main(
```

Ruff (0.8.0)

`main` is too complex (19 > 10)

(C901)